shared/actions/json/users.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
we get metadata about users all over the place but there isn't a single place we put this stuff. we get data from tracker loads, from metadata in chat, in teams. lets start to group 'user' into into one place

Reply:
putting some of this as a comment here would be helpful

Reply:
https://github.com/keybase/client/pull/10746/files#diff-f33a78f386c13b434ed1fd57b3932ad9R2 is the comment about the role of this store